test/baselines/adapter_TS_React/admin/src/lib/admin.d.ts

```diff
@@ -1,93 +0,0 @@
-declare let systemDictionary: Record<string, Record<string, string>>;
```

Comment:
Do we really not need any of the definitions, not even `adapter` and `instance`?

Reply:
No, the only thing that is required, is `window["adapterName"] = "xxx";`. Everything else is done through the `Connection` class. That's at least how Bluefox's example does it.